lib/rubocop/cop/mixin/statement_modifier.rb

```diff
@@ -58,10 +58,11 @@ def max_line_length
       end
 
       def indentation_multiplier
-        return 1 if config.for_cop('Layout/IndentationStyle')['Enabled']
+        indentation_style_config = config.for_cop('Layout/IndentationStyle')
+        return 1 if indentation_style_config['EnforcedStyle'] == 'spaces'
```

Comment:
IMHO, looks like `indentation_style_config['Enabled']` check is needed.

Reply:
Just for clarity, what do you think should happen if indentation_style_config['Enabled'] is `true`?

Reply:
Yeah, I was concerned about changing the behavior for users who have `indentation_style_config['Enabled']` is `false`.
I think it's intuitive that if a user have disabled `Enabled`, the `EnforcedStyle` will also be disabled.

Reply:
That would mean that a user or team that want to allow indentation with spaces or with tabs, without enforcing one style, has no way of specifying what the tab length is.

I don't think we have followed that principle before, that we need to check `Enabled` before using a cop's configuration from another cop, or at least it's very rare. If I search for `config.for_cop` under `rubocop/cop/` I find many of these calls, but hardly any checking of `Enabled`.